apps/frontend/src/components/user-modal/user-modal.tsx

```diff
@@ -20,19 +30,23 @@ export const UserModal: React.FC<Props> = ({
   const LOGIN = false;
   const SIGNUP = true;
 
-  const [updatePost] = useLoginMutation({
-    fixedCacheKey: 'shared-update-post',
+  const [loginTrigger, result] = useLoginMutation({
```

Comment:
```suggestion
  const [login, result] = useLoginMutation({
```
For trigger functions like this one, you can choose any name you like